apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnection.cs

```diff
@@ -523,14 +523,6 @@ private Task OpenAsyncImpl(Action transactionEnlister, CancellationToken cancell
             return ExecuteHelper.WithErrorTranslationAndProfiling(
                 async () =>
                 {
-                    // FIXME: Prevent opening instead? It's an annoyingly inconsistent state (with no _sessionPool).
```

Comment:
nit: On the method summary remove the text in bold:
(which includes acquiring a SessionPool**, unless there's no database**)

Reply:
Well spotted :) Fixed that; will wait for the integration tests to pass, then merge.